processor/batchprocessor/README.md

```diff
@@ -20,9 +20,8 @@ batch will be sent.
 - `timeout` (default = 200ms): Time duration after which a batch will be sent
 regardless of size.
 - `send_batch_max_size` (default = 0): The maximum number of items in a batch.
- This property ensures that larger batches are split into smaller units. 
- By default (`0`), there is no upper limit of the batch size. 
- It is currently supported only for the trace pipeline.
```

Comment:
Leave a message that is not supported by logs

Reply:
Done in 9b7b9da